src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -251,7 +251,7 @@ synchronized public void resourceOffers(SchedulerDriver driver, List<Offer> offe
           pendingOffers.put(offer.getId(), offer);
       }
         
-      if (!initializingCluster) {
+      if ( !initializingCluster && pendingOffers.size() >= (HDFSConstants.TOTAL_NAME_NODES + conf.getJournalNodeCount()) ) {
```

Comment:
I don't like the space after the first '('. Does this adhere to our preferred Java style?